src/lib/components/dashboard/components/Placeholder.svelte

```diff
@@ -19,24 +22,62 @@
     copiedToClipboard = true;
     isLoaderActive = false;
   };
+
+  onMount(async () => {
+    const user = await getUser();
```

Comment:
please use the user saved in local storage

Reply:
done